packetWin7/WlanHelper/WlanHelper/WlanHelper.cpp

```diff
@@ -517,48 +517,48 @@ BOOL SetWlanOperationMode(tstring strGUID, tstring strMode)
 }
 
 #define STR_COMMAND_USAGE \
-_T("WlanHelper for Npcap ") _T(WINPCAP_VER_STRING) _T(" ( http://npcap.org )\n")\
```

Comment:
Does this change do anything other than add a space before the backslash escape at the end of the line?

If so, is it necessary to support newer versions of MSVC?